package.json

```diff
@@ -23,14 +23,20 @@
     ]
   },
   "scripts": {
-    "test": "./node_modules/.bin/karma start --single-run"
```

Comment:
npm will automatically look for executables in `./node_modules/.bin/`, so no need for the long path.